DB/firebase_handler.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+from typing import List, Dict
+import firebase_admin
+from firebase_admin import credentials, db
+
+
+def initialize_db():
+    # Setting up the Firebase connection
+    cred = credentials.Certificate('../private/telegrambot-7fe4d-firebase-adminsdk-vi108-333c66edaa.json')
+    firebase_admin.initialize_app(cred, {
+        'databaseURL': 'https://telegrambot-7fe4d-default-rtdb.firebaseio.com/'
+    })
+
+
+def get_db_reference():
+    """Get a reference to the 'users' node in the database."""
+    return db.reference('users')
+
+
+
+def add_record(name: str, phone: str, symptom: str, conversation_txt: str, severity: int):
```

Comment:
When adding records, you might want to add some data validation to ensure that required fields are not empty or invalid.